dnsapi/dns_lua.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env sh
+
+# bug reports to dev@1e.ca
+
+#
+#LUA_Key="sdfsdfsdfljlbjkljlkjsdfoiwje"
+#
+#LUA_Email="user@luadns.net"
+
+LUA_Api="https://api.luadns.com/v1"
+LUA_auth=$(printf $LUA_Email:$LUA_Key | _base64)
```

Comment:
if you're talking about line 11, it may be needed at some point in dns_lua_rm() so will need to be duplicated there. let me know.